src/controllers/hotkeys/HotkeyScope.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace chatterino {
+
+enum class HotkeyScope { Tab, Split, SplitInput, Window, UserCard };
```

Comment:
split this up so each enum is on a separate line, then add a comment describing what a hotkey scope is